hackathon_site/dashboard/frontend/src/slices/store.ts

```diff
@@ -32,6 +33,7 @@ const reducers = {
     [orderReducerName]: orderReducer,
     [userReducerName]: userReducer,
     [uiReducerName]: uiReducer,
+    [teamAdminReducerName]: teamAdminReducer, // Eric Changes
```

Comment:
nice self plug haha but remove comment lol

Reply:
lol that was for me to remember where i am